metabase/metabase
# Fix getting native query for mongo returns a broken string

Fixes  [Invalid native MongoDB query](https://github.com/metabase/metabase/issues/31122)

How to verify 
1. Connect to a non-empty mongo db
2. New question from any table
3. Click "View the native query"
4. it should show a json-like mongo query
